scripts/dandi_ephys_uploads.py

```diff
@@ -22,50 +40,105 @@ def get_creds():
 
 
 def set_env():
+    """Sets the environment variable for the DANDI API key
```

Comment:
Is this function necessary? Are you calling it multiple times? 
It takes no input and return no output. 
I am concerned this add unnecessary lines to the codebase.